src/handlers/mentions.rs

```diff
@@ -50,6 +51,17 @@ pub(super) async fn parse_input(
         return Ok(None);
     }
 
+    let has_bors_commit = event.action == IssuesAction::Opened
+        && config.bors_commit_message.is_some()
+        && event
+            .issue
+            .commits(&ctx.github)
+            .await
+            .map_err(|e| log::error!("failed to fetch commits: {:?}", e))
+            .unwrap_or_default()
+            .into_iter()
+            .any(|commit| commit.commit.author.name == "bors");
```

Comment:
Does this even work for problematic rebases? As someone pointed out on Zulip, https://github.com/rust-lang/rust/pull/103918/commits does not have any bors-authored commits.